experimental/coref/project.yml

```
@@ -0,0 +1,175 @@
+
+  After that you can just run `spacy project run all`.
+
+  Note that during training you will probably see a warning like `Token indices
+  sequence length is longer than ...`. This is a rare condition that
+  `spacy-transformers` handles internally, and it's safe to ignore if it
+  happens occasionally. For more details see [this
+  thread](https://github.com/explosion/spaCy/discussions/9277#discussioncomment-1374226).
+
+  ## Using the Trained Pipeline
+
+  After you've trained the pipeline, you can load and use it like this:
+
+  ```
+  import spacy
+  nlp = spacy.load("training/coref")
+
+  doc = nlp("John Smith called from New York, he says it's raining in the city.")
+  # check the word clusters
+  print("=== word clusters ===")
+  word_clusters = [val for key, val in doc.spans.items() if key.startswith("coref_head")]
+  for cluster in word_clusters:
+      print(cluster)
+  # check the expanded clusters
+  print("=== full clusters ===")
+  full_clusters = [val for key, val in doc.spans.items() if key.startswith("coref_cluster")]
+  for cluster in full_clusters:
+      print(cluster)
+  ```
+
+  The prefixes used here are a user setting, so you can customize them for your
+  own pipelines.
+
+spacy_version: ">=3.3.0,<4.0.0"
+
+vars:
+  # XXX Change to your actual GPU ID
+  gpu_id: 0
+  # XXX fill with your local path to OntoNotes
+  ontonotes: /home/USER/ontonotes5/data
+  max_epochs: 20
+
+directories: ["assets", "corpus", "scripts", "configs", "training"]
+
+assets:
+  - dest: assets/
+    git:
+      repo: "https://github.com/explosion/conll-2012"
+      branch: "main"
+      path: ""
+    description: "CoNLL-2012 scripts and dehydrated data."
+  - dest: ${vars.ontonotes}
+    description: "Ensure you have a local copy of OntoNotes: https://catalog.ldc.upenn.edu/LDC2013T19"    
+
+workflows:
+  prep:
+    - prep-data
+    - preprocess
+  train:
+    - train-cluster
+    - prep-span-data
+    - train-span-resolver
+    - assemble
+
+  all:
+    # prep
+    - prep-data
+    - preprocess
+    # train
+    - train-cluster
+    - prep-span-data
+    - train-span-resolver
+    - assemble
+    # eval
+    - eval
+
+commands:
+  - name: "prep-data"
+    help: "Rehydrate the data using OntoNotes"
+    script: 
+      - tar xzf assets/conll-2012-development.v4.tar.gz -C assets/
+      - tar xzf assets/conll-2012-test-key.tar.gz -C assets/
+      - tar xzf assets/conll-2012-test-official.v9.tar.gz -C assets/
+      - tar xzf assets/conll-2012-train.v4.tar.gz -C assets/
+
+      # We only need the English data
+      - sed -i 's/arabic.english.chinese/english/' assets/conll-2012/v3/scripts/skeleton2conll.sh
+
+      - echo "Rehydrating data... This will take a few minutes"
+      - "bash assets/conll-2012/v3/scripts/skeleton2conll.sh -D \"${vars.ontonotes}\" assets/conll-2012"
+
+      # Concatenate the data into single files to make it easier to work with
+      - bash scripts/concat_data.sh
+    deps:
+      - ${vars.ontonotes}
```

```suggestion
    deps:
      - assets/conll-2012-development.v4.tar.gz
      - assets/conll-2012-test-key.tar.gz
      - assets/conll-2012-test-official.v9.tar.gz
      - assets/conll-2012-train.v4.tar.gz
      - assets/conll-2012/v3/scripts/skeleton2conll.sh
      - ${vars.ontonotes}
```
By adding these `deps` in specifically we avoid the generic crash
```
$ spacy project run prep-data
================================= prep-data =================================
Running command: tar xzf assets/conll-2012-development.v4.tar.gz -C assets/
tar: Error opening archive: Failed to open 'assets/conll-2012-development.v4.tar.gz'
```
if you forgot to run `spacy project assets` and instead get
```
$ spacy project run prep-data

✘ Missing dependency specified by command 'prep-data':
assets/conll-2012-development.v4.tar.gz
Maybe you forgot to run the 'project assets' command or a previous step?
```

---

Thanks, that is an improvement!